packages/fx-core/src/component/coordinator/index.ts

```diff
@@ -247,7 +250,8 @@ class Coordinator {
       } else {
         if (
           capability === CapabilityOptions.m365SsoLaunchPage().id ||
-          capability === CapabilityOptions.m365SearchMe().id
+          capability === CapabilityOptions.m365SearchMe().id ||
+          capability === CapabilityOptions.copilotM365SearchMe().id
         ) {
           inputs.isM365 = true;
```

Comment:
@jayzhang Do you know what is this setting for? Do we still need it?

Reply:
For telemetry purpose. I am not sure it's legacy or not.